client/package.json

```diff
@@ -50,9 +51,11 @@
   },
   "dependencies": {
     "@trussworks/react-uswds": "github:nathillardusds/react-uswds#nathillardusds/ssr",
+    "@types/react-helmet": "^6.1.1",
     "gatsby": "^3.4.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-helmet": "^6.1.0",
```

Comment:
as part of the security audit PR we're going to move these around; we can leave as-is for now

Reply:
Yeah, wasn't 100% sure where it needed to be. I'll move it into dev section and test.